adapter/i_o.py

```diff
@@ -694,8 +696,8 @@ def write(
         else:
             if outpath is None:
                 outpath = os.getcwd()
-
-        outpath = convert_network_drive_path(outpath, mapping=self.os_mapping)
+        if not isLocal:
```

Comment:
The method needs to be smart enough to check if local or our regular server, convert path using the identified OS, check path exists, and then return the one that is found existing. If server path exists then use it, if not then use local. However, if the output path contained something like D:\ and you are on linux, it is obvious that this is either a mounted drive or you can't write to it (it'll get written locally, but in this case we need to throw a warning).

Reply:
It does recognize local path like `my_document\output` in either OS. I probably should rename `isLocal ` to `isTemp`. `isLocal=True ` is the situation that an absolute path(local path) to the intermediate results passed.

Reply:
I found a smarter way to handle this